subprojects/docs/src/samples/signing/maven-publish/build.gradle

```diff
@@ -68,3 +68,10 @@ signing {
     sign publishing.publications.mavenJava
 }
 // END SNIPPET sign-publication
+
+
+javadoc {
+    if(JavaVersion.current().isJava9Compatible()) {
+        options.addBooleanOption('html4', true)
```

Comment:
This is added to avoid javadoc warnings on Java 9+: 

```
You have not specified the version of HTML to use.
The default is currently HTML 4.01, but this will change to HTML5
in a future release. To suppress this warning, please specify the
version of HTML used in your documentation comments and to be
generated by this doclet, using the -html4 or -html5 options.
```